projects/archiver/upload.ts

```diff
@@ -1,12 +1,23 @@
 import { s3, bucket } from './s3'
 
 export const upload = (key: string, body: {} | Buffer) => {
-    return s3
-        .putObject({
-            Body: body instanceof Buffer ? body : JSON.stringify(body),
-            Bucket: bucket,
-            Key: `${key}`,
-            ACL: 'public-read',
-        })
-        .promise()
```

Comment:
So afaict, this was hanging on to a copy of the uploaded data, which was being mapped on so we held a copy of everything in memory whenever it was uploaded to S3.